### PR TITLE
gusto auth workato connection

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: admin_settings/settings.py [hunk 1]
```diff
@@ -46,8 +46,7 @@
     'apps.users',
     'apps.bamboohr',
     'apps.orgs',
-    'apps.travelperk',
-    'apps.gusto'
```
Comment (Shwetabhk) [inline admin_settings/settings.py:50]: why remove apps.gusto from the list?

File: apps/orgs/views.py [hunk 2]
```diff
@@ -142,7 +144,8 @@ def post(self, request, *args, **kwargs):
 
             # Creating Fyle Connection In Workato
             fyle_connection_name = request.data.get('fyle_connection_name', 'Fyle Connection')
-            connection = create_connection_in_workato(fyle_connection_name, org.managed_user_id, data)
+            connection = create_connection_in_workato('Fyle Connection', org.managed_user_id, data)
```
Comment (NileshPant1999) [inline apps/orgs/views.py:147]: don't change this

Comment (DhaaraniCIT) [reply apps/orgs/views.py:147]: replaceing it

File: apps/orgs/views.py [hunk 3]
```diff
@@ -196,8 +199,7 @@ def post(self, request, *args, **kwargs):
             }
 
             # Creating Fyle Sendgrid Connection
-            sendgrid_connection_name = request.data.get('sendgrid_connection_name', 'My SendGrid account')
-            connection = create_connection_in_workato(sendgrid_connection_name, org.managed_user_id, data)
```
Comment (DhaaraniCIT) [inline apps/orgs/views.py:200]: @NileshPant1999 should i replace this as well

**Pull request merged**